src/test/regress/input/multi_load_data.source

```diff
@@ -18,9 +18,9 @@ SET citus.next_shard_id TO 290000;
 \copy orders_reference FROM '@abs_srcdir@/data/orders.2.data' with delimiter '|'
 
 \copy customer FROM '@abs_srcdir@/data/customer.1.data' with delimiter '|'
-\copy customer_append FROM '@abs_srcdir@/data/customer.1.data' with delimiter '|'
+\copy customer_append FROM '@abs_srcdir@/data/customer.1.data' with (delimiter '|', append_to_shard 360006)
 \copy nation FROM '@abs_srcdir@/data/nation.data' with delimiter '|'
 \copy part FROM '@abs_srcdir@/data/part.data' with delimiter '|'
-\copy part_append FROM '@abs_srcdir@/data/part.data' with delimiter '|'
+\copy part_append FROM '@abs_srcdir@/data/part.data' with (delimiter '|', append_to_shard 360009)
```

Comment:
Minor: we can possible use `gset` here as well if we turn `\copy` into `copy` as you did in other places, but it should be fine to leave it as is as well

Reply:
the other issue is that this copies into shards created in another file

Reply:
yeah not much we can do I guess :)